src/createNumbersGenerator.js

```diff
@@ -26,7 +26,20 @@
  * @return {number}
  */
 function createNumbersGenerator(numbers, min, max) {
-  // write code here
+  let index = 0;
+  let numbersCopy = numbers;
+  return function() {
+    if (!numbers) {
+      return undefined;
+    }
+    if (min && max) {
```

Comment:
good, but if we will receive only `max` or `min`?

you can solve this problem with another check or add default values to min and max